components/bl-alert-component/styles/index.less

```diff
@@ -106,6 +106,7 @@
 
       background-color: transparent;
       border-radius: 50%;
+      cursor: pointer;
```

Comment:
Need variable for that, also look please if we could add some variable for other controls if they exist